common/test/acceptance/tests/lms/test_lms_progress.py

```diff
@@ -0,0 +1,178 @@
+                    <p>
+                        Given the data in Table 7 <clarification>Table 7: "Example PV Installation Costs",
+                        Page 171 of Roberts textbook</clarification>, compute the ROI
+                        <clarification>Return on Investment <strong>(per year)</strong></clarification> over 20 years.
+                    </p>
+                    <numericalresponse answer="6.5">
+                        <textline label="Enter the annual ROI" trailing_text="%" />
+                    </numericalresponse>
+                </text>
+            </problem>
+        """)
+
+        course_fix.add_children(
+            XBlockFixtureDesc('chapter', 'Test Section 1').add_children(
+                XBlockFixtureDesc(
+                    'sequential', 'Test Subsection 1', metadata={'graded': True, 'format': 'Homework'}
+                ).add_children(
+                    XBlockFixtureDesc(
+                        'vertical', 'Test Vertical 1'
+                    ).add_children(XBlockFixtureDesc('problem', 'Test Problem 1', data=problem)),
+                ),
+                XBlockFixtureDesc(
+                    'sequential', 'Test Subsection 2', metadata={'graded': True, 'format': 'Homework'}
+                ).add_children(
+                    XBlockFixtureDesc(
+                        'vertical', 'Test Vertical 2'
+                    ).add_children(XBlockFixtureDesc('problem', 'Test Problem 3', data=problem)),
+                ),
+                XBlockFixtureDesc('sequential', 'Test Subsection 3').add_children(
+                    XBlockFixtureDesc(
+                        'vertical', 'Test Vertical 3'
+                    ).add_children(XBlockFixtureDesc('problem', 'Test Problem 3', data=problem)),
+                ),
+            ),
+        ).install()
+
+        # Auto-auth register for the course.
+        AutoAuthPage(
+            self.browser, username=self.USERNAME, email=self.EMAIL, course_id=self.course_id, staff=False
+        ).visit()
+
+    def check_problem(self, section, subsection, answer):
+        """
+        Opens the courseware page with a problem and passes it.
+        """
+        self.courseware.visit()
+        self.course_nav.go_to_section(section, subsection)
+        problem_page = ProblemPage(self.browser)
+        problem_page.fill_answer(answer)
+        problem_page.click_check()
+        self.assertTrue(problem_page.is_correct())
+
+    def set_assignments(self, assignments):
+        """
+        Configures assignment types for the course.
+        """
+        GradingConfigFixture(self.course_id, {'graders': assignments}).install()
+
+    def test_passing_info_table_is_hidden_when_all_passing_grades_equal_zero(self):
+        """
+        Ensures that the passing information table is hiiden when all passing grades equal 0.
+        """
+        self.set_assignments([
+            {
+                'type': 'Homework', 'passing_grade': 0, 'weight': 50,
```

Comment:
> do we need passing_grade_enabled parameter here?

Hmm, I think we need this parameter. This test was created before passing_grade_enable param was added.